airflow-core/src/airflow/ui/src/i18n/locales/he/dags.json

```diff
@@ -0,0 +1,87 @@
+{
+  "assetSchedule": "{{count}} מתוך {{total}} נכסים עודכנו",
+  "dagActions": {
+    "delete": {
+      "button": "מחק Dag",
+      "warning": "זה ימחק את כל המטא-דאטא הקשור ל-Dag, כולל ריצות ומשימות"
```

Comment:
אני חושב שתירגמנו meta data בצורה שונה במקומות האחרים?

Reply:
![image](https://github.com/user-attachments/assets/e841cf16-3883-46a3-a6f7-1af4e7a942d8)
:)

Reply:
Makes sense, we've already used "מסד-נתונים מטא" somewhere :)
I'll align to that